content/docs/0.16.x/reference/files/_index.md

```diff
@@ -1,4 +1,4 @@
 ---
-title: Files
-description: Reference pages for Keptn configuration files
+title: Configurations
```

Comment:
Or "Configuration files"?

Reply:
Johannes wanted just "Configurations" but this is, of course, open to discussion.